<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QWidget" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>467</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nombre"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Cantidad maxima de jugadores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cantJug"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="crear">
      <property name="text">
       <string>Crear Partida</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Pistas:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pista1">
        <property name="text">
         <string>Vacio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pista2">
        <property name="text">
         <string>Vacio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pista3">
        <property name="text">
         <string>Vacio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pista4">
        <property name="text">
         <string>Vacio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pista5">
        <property name="text">
         <string>Vacio</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
